src/nvim/ex_cmds_defs.h

```diff
@@ -176,7 +176,7 @@ enum {
 struct exarg {
   char_u *arg;             ///< argument of the command
   char_u *nextcmd;         ///< next command (NULL if none)
-  char_u *cmd;             ///< the name of the command (except for :make)
+  char *cmd;             ///< the name of the command (except for :make)
```

Comment:
might as well keep this aligned. The entire list doesn't need to be updated, but least this line could stay aligned.